### PR TITLE
remove redundant brackets

### PR DESCRIPTION
**Pull request by 0-v-0**

### CODE REVIEW

File: compiler/src/dmd/backend/x86/cod2.d [hunk 1]
```diff
@@ -2168,7 +2168,7 @@ void cdcom(ref CGstate cg, ref CodeBuilder cdb,elem *e,regm_t *pretregs)
     }
     tym_t tym = tybasic(e.Ety);
     int sz = _tysize[tym];
-    uint rex = (I64 && sz == 8) ? REX_W : 0;
```
Comment (thewilsonator) [inline compiler/src/dmd/backend/x86/cod2.d:2171]: this is not more easy to read as a result of this change

File: compiler/src/dmd/expressionsem.d [hunk 2]
```diff
@@ -172,7 +172,7 @@ bool expressionsToString(ref OutBuffer buf, Scope* sc, Expressions* exps)
         // char literals exp `.toStringExp` return `null` but we cant override it
         // because in most contexts we don't want the conversion to succeed.
         IntegerExp ie = e4.isIntegerExp();
-        const ty = (ie && ie.type) ? ie.type.ty : Terror;
```
Comment (thewilsonator) [inline compiler/src/dmd/expressionsem.d:175]: ditto

File: druntime/src/rt/cast_.d [hunk 3]
```diff
@@ -15,9 +15,7 @@
 module rt.cast_;
 
 extern (C):
-@nogc:
-nothrow:
-pure:
```
Comment (thewilsonator) [inline druntime/src/rt/cast_.d:20]: please leave this as is

**Review by thewilsonator — CHANGES_REQUESTED** [review]

**Review by RazvanN7 — CHANGES_REQUESTED** [review]
This is not the kind of change that we accept, mainly because it does not bring a net benefit. This sort of change is quite subjective and it can be argued both ways so instead of entering in meaningless debates we go down the conservative route and flat out reject cosmetic changes to the code.

**Pull request closed without merging**